Meta.Tests.ps1

```diff
@@ -85,17 +130,37 @@ Describe 'PowerShell DSC resource modules' {
 
             It 'all .psm1 files don''t have parse errors' {
                 $errors = @()
-                $psm1Files | %{ 
+                $psm1Files | ForEach-Object { 
                     $localErrors = Get-ParseErrors $_.FullName
                     if ($localErrors) {
                         Write-Warning "There are parsing errors in $($_.FullName)"
-                        Write-Warning ($localErrors | fl | Out-String)
+                        Write-Warning ($localErrors | Format-List | Out-String)
                     }
                     $errors += $localErrors
                 }
                 $errors.Count | Should Be 0
             }
         }
+
+        foreach ($psm1file in $psm1Files) 
+        {
+            Context "Schema Validation of $($psm1file.BaseName)" {
+
+                It 'should pass Test-xDscResource' {
+                    $result = Test-xDscResource -Name $psm1file.DirectoryName
```

Comment:
Test-xDscResource looks for the resource to test in PSModulePath. In order to make sure that we are testing development version of the module (and not the production version which user deployed e.g. to Program Files\WindowsPowershell\Modules), we should modify $env:PSModulePath here.
See https://github.com/PowerShell/xActiveDirectory/blob/dev/appveyor.yml 
lines 27-32 for example